### PR TITLE
+ 5 icons, + 2 links

### PR DESCRIPTION
**Pull request by Hamster45105**
# Description

Added 5 icons, linked 2.

### Added
Nav Garmin Connect IQ (`com.ravenfeld.nav.garmin`)
Price History (`in.pricehistory.app`)
WeMoney (`com.webank.wemoney`)
Quacker (`com.thehcj.quacker`)
酷控智能遥控 ~~ Kookong Smart Remote Control (`com.kookong.app`)

### Linked
CalcYou (`net.youapps.calcyou` → `calculator.svg`)
NumberHub (`app.myzel394.numberhub` → `unitto.svg`)

## Contributor's checklist

- [x] I followed [the Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) (upd. Jan 2024) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

### CODE REVIEW

File: svgs/quacker.svg
Comment (x9136) [inline svgs/quacker.svg:1]: The fill.

File: app/assets/appfilter.xml
Comment (x9136) [inline app/assets/appfilter.xml:1]: name="酷控智能遥控 ~~ Kookong Smart Remote Control" → name="遥控专家酷控 ~~ Kookong Smart Remote Control"
https://sj.qq.com/appdetail/com.kookong.app

**Review by x9136 — CHANGES_REQUESTED** [review]

File: app/assets/appfilter.xml
Comment (Hamster45105) [reply app/assets/appfilter.xml:1]: I got the name from [https://app.mi.com/details?id=com.kookong.app](https://app.mi.com/details?id=com.kookong.app)

Comment (x9136) [reply app/assets/appfilter.xml:1]: You can focus on the number of downloads and reviews.
sj.qq.com/appdetail/com.kookong.app — 74.9万下载 (749 000 downloads)
app.mi.com/details?id=com.kookong.app — (20次评分) (20 reviews)

I can't verify this, but I think, 750k people will leave more than 20 reviews. Given this, we can say which name will be more familiar to most.

**Pull request merged**